test/utils/fizzy_engine.cpp

```diff
@@ -23,9 +23,21 @@ std::unique_ptr<WasmEngine> create_fizzy_engine()
     return std::make_unique<FizzyEngine>();
 }
 
-void FizzyEngine::parse(bytes_view input)
+bool FizzyEngine::parse(bytes_view input)
 {
-    m_instance.module = fizzy::parse(input);
+    try
+    {
+        m_instance.module = fizzy::parse(input);
+    }
+    catch (const fizzy::parser_error&)
+    {
+        return false;
+    }
+    catch (const fizzy::instantiate_error&)
```

Comment:
So this needs to be moved to instantiate. Are we also changing instantiate to return bool? Or should it do an `abort()`?

Reply:
I think it would be good if it returns `bool`, too

Reply:
Added.